config/management/src/validator_config.rs

```diff
@@ -64,19 +64,21 @@ impl ValidatorConfig {
 
         // Step 2) Generate transaction
 
+        // TODO(davidiw): This is currently not supported
+        // let sender = self.owner_address;
+        let sender = account_address::from_public_key(&operator_key);
+
```

Comment:
@JoshLind now the comment makes sense!!!!!!

Reply:
@davidiw woooo! 🍾